Description: Drop unused but set parameter "t" which causes FTBFS with GCC 4.6
Bug-Debian: http://bugs.debian.org/625416
Author: Chow Loong Jin <hyperair@xxxxxxxxxx>
Index: tcptrack/src/TextUI.cc
===================================================================
--- tcptrack.orig/src/TextUI.cc	2011-05-03 21:16:23.970761656 +0800
+++ tcptrack/src/TextUI.cc	2011-05-03 21:16:34.720890941 +0800
@@ -119,11 +119,6 @@
 // main drawer thread loop
 void TextUI::displayer_run()
 {
-	// redraw the display this freqently...
-	struct timespec t;
-	t.tv_sec=0;
-	t.tv_nsec=DISPLAY_REFRESH;
-
 	// used for select(), to grab keyboard input when there is some.
 	fd_set fdset;
 	struct timeval tv;
